README.md

```diff
@@ -52,7 +51,7 @@ public void Should_Generate_Weekly_Recurrences_Without_End_Date()
     Assert.AreEqual(new DateTime(2014, 2, 27).Add(startDate.TimeOfDay), occurrences[3], "Date 4 should be correct");
     Assert.AreEqual(new DateTime(2014, 3, 11).Add(startDate.TimeOfDay), occurrences[4], "Date 5 should be correct");
 }
-</code></pre>
```

Comment:
Thanks!  Can you please include version bump in your PR?  Just bump if in csproj file from 3.0.0 to 3.0.1.  Thanks!